tools/syz-mutate/mutate.go

```diff
@@ -76,7 +77,11 @@ func main() {
 			fmt.Fprintf(os.Stderr, "failed to read prog file: %v\n", err)
 			os.Exit(1)
 		}
-		p, err = target.Deserialize(data, prog.Strict)
+		mode := prog.Strict
+		if !*flagStrict {
```

Comment:
Nit: it would be a bit more in line with other similar code parts in syzkaller if you default to `NonStrict` and then look at `flagStrict`.

```
mode := prog.NonStrict
if *flagStrict {
  mode = prog.Strict
}
```

Reply:
Makes sense, done.